.github/workflows/commit-message-check.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Commit Message Check'
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+jobs:
+  check-commit-message:
+    name: Check Commit Message
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Commit Type
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^\S+\((.+)\):\s(.+)$'
```

Comment:
It seems one cannot add space after parenthesis (ex: feat(bla_bla) : description))
I think it should be added in the pattern

Reply:
Why you want add a space after parenthesis ? I think we don't need complexify this pattern